jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BaseImageParameters.java

```diff
@@ -29,13 +30,32 @@
 public class BaseImageParameters {
 
   private final AuthParameters auth;
-
-  @Nullable private String image;
   @Nullable private String credHelper;
+  @Nullable private String image;
```

Comment:
nit: I see the order of existing fields has changed. Let's revert this.

Reply:
Do you mean putting image on top of credHelper?

Reply:
Yes. This is just an unnecessary change. No need to change unrelated code in this PR.

Reply:
Done !!